webapp/components/admin_console/dropdown_setting.jsx

```diff
@@ -43,5 +46,6 @@ DropdownSetting.propTypes = {
     currentValue: React.PropTypes.string.isRequired,
     handleChange: React.PropTypes.func.isRequired,
     isDisabled: React.PropTypes.bool.isRequired,
-    helpText: React.PropTypes.node.isRequired
+    helpText: React.PropTypes.node.isRequired,
+    margin: React.PropTypes.node.isRequired
```

Comment:
Margin should be `React.PropTypes.oneOf(['', 'margin'])`. A `React.PropTypes.node` can be a string, but it can also be anything that can be dropped into the virtual DOM like `<Setting />`.